sdk/storage/Azure.Storage.Files.DataLake/tests/DirectoryClientTests.cs

```diff
@@ -693,29 +693,13 @@ public async Task SetAccessControlAsync()
             using (GetNewDirectory(out DataLakeDirectoryClient directoryClient))
             {
                 // Act
-                Response<PathInfo>  response = await directoryClient.SetAccessControlAsync(acl: AccessControl);
+                Response<PathInfo>  response = await directoryClient.SetAccessControlListAsync(AccessControlList);
 
                 // Assert
                 AssertValidStoragePathInfo(response);
             }
         }
 
-        [Test]
-        public async Task SetAccessControlAsync_Error()
```

Comment:
Removed this test, since it is no longer possible to call SetAccessContolList() with an invalid access control list.